internal/pkg/deploy/pipeline.go

```diff
@@ -180,20 +242,31 @@ func (s *BitbucketSource) ConnectionName() (string, error) {
 	if err != nil {
 		return "", fmt.Errorf("parse owner and repo to generate connection name: %w", err)
 	}
+	return formatConnectionName(owner, repo), nil
+}
+
+func (s *GitHubSource) ConnectionName() (string, error) {
```

Comment:
doc comments :)

Reply:
Ohhh, I had them on lines 229-232 for the whole section, but I see why there should be comments immediately preceding the methods. Adding!